console/frontend/pom.xml

```diff
@@ -34,6 +34,11 @@
 			<artifactId>jakarta.servlet-api</artifactId>
 			<scope>provided</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+		</dependency>
```

Comment:
Aparently needed because of surefire... It doesn't need to do anything but has to be on the classpath.